htdocs/resources/views/alert_form.blade.php

```diff
@@ -106,123 +106,8 @@
-        prefetch: {
-            ttl: 0,
-            url: window.location.origin + '/typeahead/listcolumn/es_index/alerts'
-        }
-    });
-
-    $('#prefetch_indexes .typeahead').typeahead(null, {
-        name: 'es_indexes',
-        source: es_indexes,
-        limit: 10,
-    });
-
-    var es_types = new Bloodhound({
-        datumTokenizer: Bloodhound.tokenizers.whitespace,
-        queryTokenizer: Bloodhound.tokenizers.whitespace,
-        // url points to a json file that contains an array of country names, see
-        // https://github.com/twitter/typeahead.js/blob/gh-pages/data/countries.json
-        prefetch: {
-            ttl: 0,
-            url: window.location.origin + '/typeahead/listcolumn/es_type/alerts'
-        }
-    });
-
-    $('#prefetch_types .typeahead').typeahead(null, {
-        name: 'es_types',
-        source: es_types,
-        limit: 10,
-    });
-
-    var es_datetime_fields = new Bloodhound({
-        datumTokenizer: Bloodhound.tokenizers.whitespace,
-        queryTokenizer: Bloodhound.tokenizers.whitespace,
-        // url points to a json file that contains an array of country names, see
-        // https://github.com/twitter/typeahead.js/blob/gh-pages/data/countries.json
-        prefetch: {
-            ttl: 0,
-            url: window.location.origin + '/typeahead/listcolumn/es_datetime_field/alerts'
-        }
-    });
-
-    $('#prefetch_es_date_time_field .typeahead').typeahead(null, {
-        name: 'es_datetime_fields',
-        source: es_datetime_fields,
-        limit: 10,
-    });
-
-
-    var alert_email_senders = new Bloodhound({
-        datumTokenizer: Bloodhound.tokenizers.whitespace,
-        queryTokenizer: Bloodhound.tokenizers.whitespace,
-        // url points to a json file that contains an array of country names, see
-        // https://github.com/twitter/typeahead.js/blob/gh-pages/data/countries.json
-        prefetch: {
-            ttl: 0,
-            url: window.location.origin + '/typeahead/listcolumn/alert_email_sender/alerts'
-        }
-    });
-
-    $('#prefetch_alert_email_sender .typeahead').typeahead(null, {
-        name: 'alert_email_senders',
-        source: alert_email_senders,
-        limit: 10,
-    });
-
-    var alert_email_recipients = new Bloodhound({
-        datumTokenizer: Bloodhound.tokenizers.whitespace,
-        queryTokenizer: Bloodhound.tokenizers.whitespace,
-        // url points to a json file that contains an array of country names, see
-        // https://github.com/twitter/typeahead.js/blob/gh-pages/data/countries.json
-        prefetch: {
-            ttl: 0,
-            url: window.location.origin + '/typeahead/listcolumn/alert_email_recipient/alerts'
-        }
-    });
-
-    $('#prefetch_alert_email_recipient .typeahead').typeahead(null, {
-        name: 'alert_email_recipients',
-        source: alert_email_recipients,
-        limit: 10,
-    });
 
-    </script>
+    <meta type="module" name="alert-form"/>
```

Comment:
Instead of writing untestable inline javascipt I'm declaring a module with a meta tag. Then with Jquery I'm compiling the module <Angular style> from the DOM.  I used browserify to make CommonJS modules instead and I'm handling initialisation in there.